<launch>

<node name="stero_bottom" pkg="prosilica_stereo" type="stereo_node" output="screen">
    <param name="master_ip" type="str" value="192.168.1.100" />
    <param name="slave_ip" type="str" value="192.168.2.100" />
    <param name="master_frameid" value="left_camera" />
    <param name="slave_frameid" value="right_camera" />
  </node>
</launch>
